operator/src/main/java/oracle/kubernetes/operator/work/Packet.java

```diff
@@ -15,6 +15,7 @@
   private final ConcurrentMap<String, Component> components =
       new ConcurrentHashMap<String, Component>();
   private final ConcurrentMap<String, Object> delegate = new ConcurrentHashMap<String, Object>();
+  private boolean isDynamicUpdate = false;
```

Comment:
Okay, this is the wrong spot for this.  Reminder to myself to relocate as a property.

Reply:
We can actually remove this for v1

Reply:
This is not require for v1

Reply:
Okay, I'll take care of it unless you tell me you've already done it